docs/fermi_hubbard/experiment_example.ipynb

```diff
@@ -15,7 +68,90 @@
     "id": "9ec578c1796c"
    },
    "source": [
-    "This notebook demonstrates how to execute a single instance of Fermi-Hubbard experiment on Google processor."
+    "This notebook demonstrates how to define, execute, and plot the results of a single instance of [the Fermi-Hubbard experiment](https://arxiv.org/abs/2010.07965). We show how to run the experiment using a Cirq simulator and a quantum processor through Google's Quantum Computing Service.\n",
+    "\n",
+    "The Fermi-Hubbard model on a one-dimensional lattice of $L$ sites with open boundary conditions is defined by the Hamiltonian\n",
+    "\n",
+    "$$\n",
+    "H = - J \\sum_{j = 1}^{L - 1} \\sum_{\\nu = \\uparrow, \\downarrow} c_{j, \\nu}^\\dagger c_{j + 1, \\nu} + \\text{h.c.} + U \\sum_{j = 1}^{L} n_{j, \\uparrow} n_{j, \\downarrow} + \\sum_{j = 1}^{L} \\sum_{\\nu = \\uparrow, \\downarrow} \\epsilon_{j, \\nu} n_{j, \\nu} \n",
```

Comment:
What we are really supporting in this experiment is an extended Hamiltonian with a next-neighbor interaction terms V = n_{j, \\nu} n_{j + 1, \\nu}. This is mentioned on page 2 of the Fermi-Hubbard paper. We didn't include that in the equation (1) of the paper because of the clarity.

I think, this codebase is not only the Fermi-Hubbard paper and it might be good to include this full Hamiltonian description. This is not the usual Fermi-Hubbard model though. This V term is used in "Parasitic-controlled-phase" section below for example.

Reply:
The code also supports site-dependent U, J and V coefficients (something like U_j, J_j, and V_j). In the paper we simplified it because we only used site-dependence for V term but this codebase supports more general Hamiltonians.

Reply:
Thanks a lot @mrwojtek for these comments. I added a note about the parasitic controlled phase as well as supporting site-dependent coefficients. I kept the equation of the Hamiltonian the same as per the paper, but it's also possible to include the cphase-induced interaction term explicitly. Let me know what you think.

Reply:
I think it's perfectly fine to put all this information in the text and not in the main equation.

I tend to think about the V term more like an ability to simulate extended physics, less as a necessary evil caused by our hardware, i.e., this is not only used for simulation on classical computer. If you introduce non-zero V terms and run it on a hardware, the circuits will have nearest-neighbor CPHASE gates and this extended Hubbard model will be actually simulated on a quantum device. Could you reformulate this more in a way that this is an extension to the model that we support, with less impact on the details caused by the device behavior?